modules/market/src/main/java/com/opengamma/strata/market/param/YearMonthDateParameterMetadata.java

```diff
@@ -69,7 +69,6 @@
    * @return the parameter metadata based on the year-month
    */
   public static YearMonthDateParameterMetadata of(LocalDate date, YearMonth yearMonth) {
-    ArgChecker.notNull(date, "date");
```

Comment:
I think this should be left in - when one arg checker is needed, I would usually check everything